src/uv-common.c

```diff
@@ -225,6 +225,30 @@ int uv_ip6_name(const struct sockaddr_in6* src, char* dst, size_t size) {
   return uv_inet_ntop(AF_INET6, &src->sin6_addr, dst, size);
 }
 
+int uv_if_indextoname(unsigned int ifindex, char* ifname) {
```

Comment:
Other functions we have which write to a buffer take a second length parameter. I'd like us to do the same here. It would be an in-out parameter. See this similar API: http://docs.libuv.org/en/v1.x/fs_poll.html#c.uv_fs_poll_getpath

Please make sure the terminating NULL byte is handled consistently.